gen_bot_db.sh

```diff
@@ -29,6 +29,26 @@ verify_inputs() {
   fi
 }
 
+request_access_token() {
+  client_id=$(pass api.eclipse.org/client_id)
+  client_secret=$(pass api.eclipse.org/client_secret)
+  access_token=$(curl -s --request POST \
+    --url 'https://accounts.eclipse.org/oauth2/token' \
+    --header 'content-type: application/x-www-form-urlencoded' \
+    --data 'grant_type=client_credentials' \
+    --data "client_id=${client_id}" \
+    --data "client_secret=${client_secret}" \
+    --data 'scope= eclipsefdn_view_all_profiles' | jq .access_token | tr -d '"')
```

Comment:
Use `jq -r` for raw output instead of json value.
```suggestion
    --data 'scope= eclipsefdn_view_all_profiles' | jq -r '.access_token')
```

Reply:
Fixed.